src/ReactiveDomain.Core/ReactiveDomain.Core.csproj

```diff
@@ -7,5 +7,6 @@
   <PropertyGroup>
     <TargetFrameworks>$(LibTargetFrameworks)</TargetFrameworks>
     <RootNamespace>ReactiveDomain</RootNamespace>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
These should be conditional on build configuration -- we probably don't want packages generated by a Debug build.

Reply:
Yeah there may be some extra attribute for that. Seems ok for packages to be generated though as long as they dont get pushed to the feed. I have to add logic to the .yml file for that which is what im working on now. 

Note: I need an open pull request to test the changes so I am continuously changing. I will give update when this pr is final